### ISSUE TITLE
Build icon in README links to incorrect page

### ISSUE THREAD

**Issue by matthiaskoenig** [draeger-lab/SBSCL#33]
The tests are currently failing for the latest commits, but the build icon links to another repository in the README which is passing
```
[![Build Status](https://travis-ci.org/shalinshah1993/SBSCL.svg?branch=master&style=plastic)](https://travis-ci.org/shalinshah1993/SBSCL)
```
[![Build Status](https://travis-ci.org/shalinshah1993/SBSCL.svg?branch=master&style=plastic)](https://travis-ci.org/shalinshah1993/SBSCL)
This is confusing and incorrect. Link to correct travis results page.

**Comment by matthiaskoenig** [comment]
@hemilpanchiwala could you also have a look at this one. Is a minor fix in the link in the Readme.

**Comment by hemilpanchiwala** [comment]
@matthiaskoenig, I will fix it with the same PR solving issue #34.
